src/main/java/org/testng/reporters/jq/Main.java

```diff
@@ -18,10 +18,11 @@
 
 public class Main implements IReporter {
   private static final String[] RESOURCES = new String[] {
-    "jquery-1.7.1.min.js", "testng-reports.css", "testng-reports.js",
-    "passed.png", "failed.png", "skipped.png", "navigator-bullet.png",
-    "bullet_point.png", "collapseall.gif"
```

Comment:
I think this will break the report generation. If you look [here](https://github.com/cbeust/testng/blob/master/src/main/java/org/testng/reporters/jq/Main.java#L74-L87), you will notice that TestNG iterates the array `RESOURCES` and copies the files using the name to the output directory.
Since now we are renaming the file names to be prefixed with `org/testng/` the files that are created by `Files.copyFile(is, new File(m_outputDirectory, fileName));` will also have their file name prefixed with `org/testng`.

I think this will cause the reports to be broken. If we are going to be altering the location of the resources, then I believe we should also take care of altering the logic [here](https://github.com/cbeust/testng/blob/master/src/main/java/org/testng/reporters/jq/Main.java#L74-L87)

For the sake of ease, I am including the corresponding code which will be impacted and needs to be changed.

```java
    try {
      try (InputStream header = getClass().getResourceAsStream("/header")) {
        if (header == null) {
          throw new RuntimeException("Couldn't find resource header");
        }
        for (String fileName : RESOURCES) {
          try (InputStream is = getClass().getResourceAsStream("/" + fileName)) {
            if (is == null) {
              throw new AssertionError("Couldn't find resource: " + fileName);
            }
            Files.copyFile(is, new File(m_outputDirectory, fileName));
          }
        }
        all = Files.readFile(header);
        Utils.writeUtf8File(m_outputDirectory, "index.html", xsb, all); 
      }
    } catch (IOException e) {
      // TODO Auto-generated catch block
      e.printStackTrace();
    }
```